src/Simulations/time_step_wizard.jl

```diff
@@ -67,7 +70,13 @@ function TimeStepWizard(FT=Float64;
                         min_Δt = 0.0,
                         cell_advection_timescale = cell_advection_timescale,
                         cell_diffusion_timescale = infinite_diffusion_timescale)
-
+    
+    # check if user gave max_change or min_change values that are invalid
+    if min_change >= 1.0 || max_change <= 1.0
+        throw(ArgumentError("Invalid value: min_change should be < 1 and max_change should be > 1. " *
+                            "You provided min_change = $min_change and max_change = $max_change."))
+    end
```

Comment:
is this simpler and cleaner?
```suggestion
    min_change ≥ 1 && throw(ArgumentError("min_change must be < 1"))

    max_change ≥ 1 && throw(ArgumentError("max_change must be > 1"))
```